fpga/pulpissimo-zcu106/README.md

```diff
@@ -0,0 +1,57 @@
+# PULPissimo on the Xilinx ZCU104 Board
```

Comment:
Please update this README file to be inline with the port (i.e. update default frequencies, titles etc.)